ingestion/functions/README.md

```diff
@@ -71,13 +71,18 @@ ls -l /usr/bin/python*
 modules installed, e.g. via:
 
 ```shell
+# In each parsing's subdir:
 python3.6 -m pip install -r requirements.txt
+# In the /ingestion/functions (necessary to run unit tests).
+python3.6 -m pip install -r ci-requirements.txt
```

Comment:
Was this required for you? This is supposed to be so that GitHub Actions doesn't need to install individual requirements files.

Reply:
Yes otherwise I would miss the moto requirements `moto==1.3.14`